src/dashboard/src/main/management/commands/rebuild_aip_index_from_storage_service.py

```diff
@@ -114,109 +117,72 @@ def handle(self, *args, **options):
         logging.getLogger("elasticsearch").setLevel(logging.ERROR)
         logging.getLogger("archivematica.common").setLevel(logging.ERROR)
 
-        # Setup es_client and delete indices if required.
-        es_client = setup_es_for_aip_reindexing(self, options["delete_all"])
-
         # Create temporary directory for downloaded METS files.
         temp_dir = tempfile.mkdtemp()
 
-        # Set pipeline UUID to use for filtering packages to reindex.
         pipeline_uuid = options["pipeline"]
 
         delete_before_reindexing = False
-        if options["delete"] is True:
+        if options["delete"]:
             delete_before_reindexing = True
 
-        # Index a single package and return if uuid option is enabled.
         if options["uuid"]:
```

Comment:
Honestly I went back and forth on this because I didn't want to unnecessarily limit users, but ultimately I think you're right that the impact of that mistake outweighs the remote possibility that someone would want to delete their entire index and reindex only one package (especially since I can't think of a use case for the latter). I'll make the change before merging.